install/tools/ipa-ca-install

```diff
@@ -106,6 +125,33 @@ def get_dirman_password():
         "Directory Manager (existing master)", confirm=False, validate=False)
 
 
+def reset_subject_base_config(dm_password, realm, subject_base):
+    """(Re)set CA subject base config in all relevant places."""
+    # LDAP
+    set_subject_base_in_config(realm, dm_password, subject_base)
+
+    # sysupgrade
+    sysupgrade.set_upgrade_state('certmap.conf', 'subject_base', subject_base)
```

Comment:
IMO the scope is not big (only 4 places including this one need an update), but OK.